src/components/CustomCollection/CustomCollection.tsx

```diff
@@ -172,16 +187,23 @@ const CustomCollection = ({
               <Label htmlFor="bookmarkId" className="usa-sr-only">
                 Select existing link
               </Label>
+
               <ComboBox
                 id="bookmarkId"
                 name="bookmarkId"
                 className={styles.addLinkComboBox}
-                options={urlOptions}
+                options={[...urlOptions, addCustomLinkOption]}
                 onChange={handleSelectChange}
                 ref={linkInput}
+                customFilter={{
+                  filter: '.*({{query}}|custom|{{query}},custom).*',
```

Comment:
This is the regex that makes it all happen. It will return any matches to the user's input, and always return the `Add Custom Link` option, which has the value `custom`

Reply:
question: Did you create a chore to document this in react-uswds? Could you link it to the PR if you did?

Reply:
will do -- on my todo list for tomorrow